frontend/public/components/utils/volume-kebab.tsx

```diff
@@ -0,0 +1,58 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import { asAccessReview, Kebab, KebabOption } from './index';
+import { K8sKind, K8sResourceKind, K8sResourceKindReference } from '../../module/k8s';
+import { connectToModel } from '../../kinds';
+import { RowVolumeData } from '../volumes-table';
+import { removeVolumeModal} from '../modals';
+import { Tooltip } from './tooltip';
+
+export const VolumeKebabMenu: VolumeKebabFactory = {
+  RemoveVolume: (kind, obj, volume) => {
+    const label = 'Remove Volume';
+    let callback = undefined;
+    let tooltip, msg;
+    const accessReview = asAccessReview(kind, obj, 'patch');
+    if (kind.kind === 'Pod') {
+      tooltip = 'Pod volumes cannot be removed';
```

Comment:
I know @bmignano suggested this, but I'd argue it's inconsistent with other parts of console. If you can never perform an action on a resource, we don't show the action. I'd remove the special logic here and hide the kebab entirely.

Reply:
That makes sense, I'm okay with removing the kebab 👍

Reply:
Pod will not have a kebab menu.